lms/static/js/student_account/views/LoginView.js

```diff
@@ -102,7 +103,11 @@
             },
 
             saveError: function( error ) {
-                this.errors = ['<li>' + error.responseText + '</li>'];
+                if(error.status === 500){
```

Comment:
This can be simplified:

```javascript
var msg = error.responseText;
if(error.status == 500){
  msg = gettext('Something went wrong.');
}
this.errors = ['<li>' + msg + '</li>'];

```